### PR TITLE
Update to Keycloak 1.5.x

### PR DESCRIPTION
**Pull request by abstractj**
See: https://issues.jboss.org/browse/AGPUSH-1539

It depends on: https://github.com/aerogear/aerogear-parent/pull/53

To test it:

1. mvn clean install on aerogear-parent
2. Follow the normal steps to deploy unifiedpush server

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <groupId>org.jboss.aerogear</groupId>
         <artifactId>aerogear-parent</artifactId>
         <!-- NOTE: if updated for version of UPS dependencies, please update the 'aerogear.bom.version' as well -->
-        <version>0.2.17</version>
+        <version>0.2.17-SNAPSHOT</version>
```
Comment (matzew) [inline pom.xml]: sorry - this is my fault. it should be 0.2.18-snapshot - as 0.2.17 is alreay out on maven-central

I just forgot to push the update to master (on parent) last week :/

Comment (abstractj) [inline pom.xml]: @matzew updated, let me know if anything else is required.

**Comment by abstractj** [comment]
@matzew updated

**Comment by matzew** [comment]
Thanks, @abstractj - I have tested this on WF-9.

Getting a parent release out now

**Comment by matzew** [comment]
Also tested w/ EAP 6.4.4

**Comment by matzew** [comment]
@abstractj looks like the 0.2.18 parent is now out on maven central

**Comment by abstractj** [comment]
@matzew updated, I think we're good to go

**Comment by matzew** [comment]
ok, let me do a final test :)

On Fri, Dec 4, 2015 at 2:36 PM, Bruno Oliveira <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> updated, I think we're good to go
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/646#issuecomment-161969858>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by matzew** [comment]
:tada: this landed on master branch

:rocket:

**Pull request closed without merging**